prov/sockets/src/sock_cntr.c

```diff
@@ -420,7 +421,11 @@ static int sock_cntr_verify_attr(struct fi_cntr_attr *attr)
 	switch (attr->wait_obj) {
 	case FI_WAIT_NONE:
 	case FI_WAIT_UNSPEC:
+#ifndef _WIN32
 	case FI_WAIT_MUTEX_COND:
+#else /* _WIN32 */
+	case FI_WAIT_CRITSEC_COND:
+#endif /* _WIN32 */
```

Comment:
Is there a rationale behind why some of the `case FI_WAIT_CRITSEC_COND` lines in this change are guarded by an `#ifdef` and some are not?